test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -1376,6 +1418,120 @@ describe('33acrossBidAdapter:', function () {
         });
       });
     });
+
+    context('when SRA mode is enabled', function() {
+      it('builds a single request with multiple imps corresponding to each group {siteId, productId}', function() {
+        sandbox.stub(config, 'getConfig').callsFake(() => {
+          return {
+            enableSRAMode: true
+          }
+        });
+
+        const bidRequests = new BidRequestsBuilder()
+          .addBid()
+          .addBid({
+            bidId: 'b3',
+            adUnitCode: 'div-id',
+            params: {
+              siteId: 'sample33xGUID123456780',
+              productId: PRODUCT_ID
```

Comment:
for consistency purposes, maybe replace PRODUCT_ID with 'siab' or viceversa (replace 'siab' hardcoded strings below with PRODUCT_ID).

it also seems that this uint test only covers the different 'siteId' case, but I am not seeing the different productId case being covered.